holesky/run.sh

```diff
@@ -52,14 +54,29 @@ listQuorums() {
     --volume "${NODE_ECDSA_KEY_FILE_HOST}":/app/operator_keys/ecdsa_key.json \
     --volume "${NODE_BLS_KEY_FILE_HOST}":/app/operator_keys/bls_key.json \
     --volume "${NODE_LOG_PATH_HOST}":/app/logs:rw \
-    ghcr.io/layr-labs/eigenda/opr-nodeplugin:release-0.6.1 \
+    "$node_plugin_image" \
     --ecdsa-key-password "$NODE_ECDSA_KEY_PASSWORD" \
     --bls-key-password "$NODE_BLS_KEY_PASSWORD" \
     --socket "$socket" \
     --operation list-quorums \
     --quorum-id-list 0
 }
 
+updateSocket() {
+  # we have to pass a dummy quorum-id-list as it is required by the plugin
+  docker run --env-file .env \
+    --rm \
+    --volume "${NODE_ECDSA_KEY_FILE_HOST}":/app/operator_keys/ecdsa_key.json \
```

Comment:
will it fail if the operator doesn't use ECDSA key?

Reply:
yes, the docker command will fail. I will be revamping this whole thing later - but yes it will fail if we provide empty location